contracts/Cruize.sol

```diff
@@ -16,85 +20,88 @@ contract Cruize is Proxy, AaveV2Wrapper {
 
     /**
      * @notice initialize will initialize the crContract.
-     * @param _cruize_wallet address -  Cruize Wallet address.
-     * @param _crContract address -  ERC20Upgradeable Contract.
+     * @param _dydxWallet Cruize Wallet address.
+     * @param _crContract CrToken tokens implementation for proxy
      */
 
-    function initialize(address _cruize_wallet, address _crContract)
+    function initialize(address _dydxWallet, address _crContract)
         external
         initializer
     {
         __Ownable_init();
         __ReentrancyGuard_init();
         require(_crContract != address(0));
-        require(_cruize_wallet != address(0));
+        require(_dydxWallet != address(0));
+        toTreasury = 1000; // 10% = 1000 
         crContract = _crContract;
-        cruize_wallet = _cruize_wallet;
+        dydxWallet = _dydxWallet;
     }
 
     /**
-     * @notice createToken  will Clone CRTokenUpgradeable (ERC20 token).
-     * @param name name of   ERC20Upgradeable Contract.
-     * @param symbol symbol of   ERC20Upgradeable Contract.
-     * @param decimal decimal value of ERC20Upgradeable Contract.
-     * @dev incase of ETH  , we will take token address of WETH becuase
-     * ETH is a native token of Ethereum so we don't have token address for ETH.
+     * @notice createToken will deploy CrToken proxies
+     * @param _name name of CrToken.
+     * @param _symbol symbol of CrToken.
+     * @param _reserve underlaying asset
+     * @param _tokenOracle underlaying asset oracle
+     * @param _decimal decimal value of CrToken.
      */
 
     function createToken(
-        string memory name,
-        string memory symbol,
-        address reserve,
-        address tokenOracle,
-        uint8 decimal
+        string memory _name,
+        string memory _symbol,
+        address _reserve,
+        address _tokenOracle,
+        uint8 _decimal,
+        uint24 _fee
     ) external onlyOwner nonReentrant {
-        if (lpTokens[reserve] != address(0)) revert AssetAlreadyExists();
-        if (tokenOracle == address(0) || reserve == address(0))
+        if (crTokens[_reserve] != address(0)) revert AssetAlreadyExists();
+        if (_tokenOracle == address(0) || _reserve == address(0))
             revert ZeroAddress();
 
-        ILPtoken crToken = ILPtoken(createClone(crContract));
-        lpTokens[reserve] = address(crToken);
-        oracles[reserve] = tokenOracle;
-        crToken.initialize(name, symbol, decimal);
-        emit CreateToken(address(crToken), name, symbol, decimal);
+        ICRToken crToken = ICRToken(createClone(crContract));
+        crTokens[_reserve] = address(crToken);
+        oracles[_reserve] = _tokenOracle;
+        fees[_reserve][WETH] = _fee;
```

Comment:
can you let me know what fee you are adding here to the mapping

Reply:
Here we will add uniswap v3 pool fee